_data/victoriateo/victoriateo.md

```diff
@@ -0,0 +1,7 @@
+---
+name: Victoria Teo
+institution: National University of Singapore
+profile_pic: photo6174763929457240098.jpg
```

Comment:
Incorrect file name, please do one of the following:
1. Change `photo6174763929457240098.jpg` to `photo6174763929457240098 (1).jpg` in the markdown file.
2. Rename `photo6174763929457240098 (1).jpg` to `photo6174763929457240098.jpg`.
I'd suggest you rename the file!